tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolAtCursorTest.java

```diff
@@ -207,36 +208,39 @@ public void testQuotedIdentifiers(int line, int column, String expSymbolName) {
     @DataProvider(name = "QuotedIdentifierProvider")
     public Object[][]  getQuotedIdentifierPositions() {
         return new Object[][]{
-                {18, 5, "'record"},
-                {19, 8, "'float"},
-                {22, 23, "'from"},
-                {23, 28, "'from"},
-                {26, 9, "'function"},
-                {30, 11, "'function"},
-                {31, 7, "'if"},
-                {32, 11, "'if"},
-                {33, 7, "nonReservedVar"},
-                {39, 4, "'any"},
-                {40, 13, "'any"},
-                {43, 11, "w1"},
-                {45, 19, "'worker"},
-                {52, 11, "'worker"},
-                {58, 18, "'string"},
-                {60, 10, "floatNum"},
-                {62, 25, "'foreach"},
-                {63, 13, "'string"},
-                {64, 20, "'int"},
-                {68, 48, "'string"},
-                {70, 13, "'from"},
-                {77, 27, "Example"},
-                {79, 11, "'check"},
-                {84, 7, "'int"},
-                {85, 8, "'from"},
-                {85, 19, "'from"},
-                {87, 4, "Example"},
-                {87, 13, "'new"},
-                {88, 4, "'new"},
-                {88, 9, "'anydata"}
+                {18, 5, "'record", "record"},
+                {19, 8, "'float", "float"},
+                {22, 23, "'from", "from"},
+                {23, 28, "'from", "from"},
+                {26, 9, "'function", "function"},
+                {30, 11, "'function", "function"},
+                {31, 7, "'if", "if"},
+                {32, 11, "'if", "if"},
+                {33, 7, "'nonReservedVar", "nonReservedVar"},
+                {39, 4, "'any", "any"},
+                {40, 13, "'any", "any"},
+                {43, 11, "'w1", "w1"},
+                {45, 19, "'worker", "worker"},
+                {52, 11, "'worker", "worker"},
+                {58, 18, "'string", "string"},
+                {60, 10, "'floatNum", "floatNum"},
+                {62, 25, "'foreach", "foreach"},
+                {63, 13, "'string", "string"},
+                {64, 20, "'int", "int"},
+                {68, 48, "'string", "string"},
+                {70, 13, "'from", "from"},
+                {78, 27, "'Example", "Example"},
+                {80, 11, "'check", "check"},
+                {85, 7, "'int", "int"},
+                {86, 8, "'from", "from"},
+                {86, 19, "'from", "from"},
+                {88, 4, "'Example", "Example"},
+                {88, 13, "'new", "new"},
+                {89, 4, "'new", "new"},
+                {89, 9, "'anydata", "anydata"},
+                {90, 11, "'foo", "foo"},
+                {91, 15, "'string", "string"},
+                {92, 12, "'intVal", "\\u{69}ntVal"}
```

Comment:
Shall we add a few more unicode identifier related tests like this? Different combinations